Panel updates

emullaney opened the pull request:
Updates per this [slack convo](https://djangoconus.slack.com/archives/G19A3L25P/p1723579396230849)

emullaney committed:
Panel updates

emullaney committed:
Update title

drewbrew commented on src/_content/schedule/talks/panel-discussion-details-tba.md:
```diff
@@ -3,10 +3,30 @@ category: talks
 - lacey-henschel
 room: Junior Ballroom
-title: Panel Discussion (details TBA)
+title: Panel Discussion
```
```suggestion
title: Panel Discussion: Open, friendly, and welcoming: on the history and future of mentoring in Django
```

emullaney replied:
I caught and updated this right after I asked for review!

drewbrew replied:
Nice! I'd like to keep the fact that it's a panel in the title somehow, but other than that it's great

drewbrew commented on src/_content/schedule/talks/panel-discussion-details-tba.md:
```diff
@@ -3,10 +3,30 @@ category: talks
 difficulty: All
 end_datetime: 2024-09-25 17:15:00-04:00
-permalink: /talks/panel-discussion-details-tba/
+permalink: /talks/panel-discussion-details/
```
```suggestion
permalink: /talks/panel-discussion-mentoring-django/
```

drewbrew reviewed (approved):
LGTM, just adding in the title

emullaney committed:
Update src/_content/schedule/talks/panel-discussion-details-tba.md

Co-authored-by: Drew Winstel <drewbrew@users.noreply.github.com>

tim-schilling commented:
@jcjudkins I think we can either do a blog post about this or announce it via social media. I think this task was assigned to you in our latest meeting.

tim-schilling reviewed (approved):
The panel markdown file looks good! It might be worth changing the name. Maybe `panel-discussion-django-communities.md` or something like that.

Please wait for Jason to comment before merging in case there's an associated announcement.

emullaney committed:
Rename panel-discussion-details-tba.md to panel-discussion-django-communities.md

emullaney committed:
Add "Panel" to title

emullaney commented:
@tim-schilling @drewbrew 
- I've updated the title to include the word panel
- I've updated the filename (didn't know I could do that all inside of github, yay!)
- I won't merge this since we're waiting to coordinate with @jcjudkins via communications team

jcjudkins commented:
> @jcjudkins I think we can either do a blog post about this or announce it via social media. I think this task was assigned to you in our latest meeting.

@tim-schilling Talked with Rachell about this yesterday. Working on a draft!

carltongibson commented on src/_content/schedule/talks/panel-discussion-django-communities.md:
```diff
@@ -0,0 +1,32 @@
+- **Keanya Phelps**, Django Girls organizer and participant, mentor in Djangonaut Space
+- **Afi Maame Dufie**, involved in Django Girls, PyLadies, and Black Python Devs
+- **Sarah Boyce**, Django Fellow, co-founder and mentor in Djangonaut Space
+- **Carlton Gibson**, former Django Fellow, mentor in Djangonaut Space, contributor to Google Summer of Code and Django Girls
```
The GSoC bit didn't feel right. (I never was a GSoC _contributor_...) Maybe: 

```suggestion
- **Carlton Gibson**, former Django Fellow, mentor in Djangonaut Space, mentor and coordinator of Django's efforts in Google Summer of Code and Django Girls coach and advisory board member.
```

drewbrew committed:
✏️ Update Carlton's bio

Co-authored-by: Carlton Gibson <carlton@noumenal.es>

drewbrew commented on src/_content/schedule/talks/panel-discussion-django-communities.md:
```diff
@@ -0,0 +1,32 @@
+- **Keanya Phelps**, Django Girls organizer and participant, mentor in Djangonaut Space
+- **Afi Maame Dufie**, involved in Django Girls, PyLadies, and Black Python Devs
+- **Sarah Boyce**, Django Fellow, co-founder and mentor in Djangonaut Space
+- **Carlton Gibson**, former Django Fellow, mentor in Djangonaut Space, mentor and coordinator of Django's efforts in Google Summer of Code and Django Girls coach and advisory board member.
```
```suggestion
- **Carlton Gibson**, former Django Fellow, mentor in Djangonaut Space, mentor and coordinator of Django's efforts in Google Summer of Code, and Django Girls coach and advisory board member.
```

Just adding in an Oxford comma for consistency

drewbrew committed:
✏️ Add an oxford comma

mtrythall committed:
Adds docs for sponsor list

drewbrew commented on src/_content/schedule/talks/panel-discussion-django-communities.md:
```diff
@@ -0,0 +1,32 @@
+- **Afi Maame Dufie**, involved in Django Girls, PyLadies, and Black Python Devs
+- **Sarah Boyce**, Django Fellow, co-founder and mentor in Djangonaut Space
+- **Carlton Gibson**, former Django Fellow, mentor in Djangonaut Space, mentor and coordinator of Django's efforts in Google Summer of Code, and Django Girls coach and advisory board member.
+- **Ron Maravanyika**, co-founder of ZimboPy, current chair for PyCon Zimbabwe, and Django Girls organizer
```
```suggestion
- **Velda Kiara**,  DEFNA director, former [AnitaB.org](http://anitab.org/) Kenya Community Lead, former Open Source Community Africa Nairobi organizer and contributor, participant in Djangonaut Space, and current contributor and leader of Black Python Devs
```
Replacing Ron with Velda on the panel

drewbrew committed:
✏️ Replace Ron with Velda

jcjudkins committed:
Panel Discussion 1st Draft (#97)

* Panel Discussion 1st Draft

* updated with new information

* Apply suggestions from code review
---------

Co-authored-by: Tim Schilling <schillingt@better-simple.com>
Co-authored-by: Drew Winstel <drewbrew@users.noreply.github.com>

tim-schilling committed:
Minor updates to get things to build.

tim-schilling committed:
Fix the date and title for the panel discussion post.
